test/qa/lib/allowlist.py

```diff
@@ -37,6 +44,7 @@ def add_ports_to_allowlist(ports_list: list[Port], allowlist_alias="allowlist"):
             port_value = port.value
             expected_message = MSG_ALLOWLIST_PORT_ADD_SUCCESS % (port_value, port.protocol)
 
+        allowlist_alias = random.choice(ALLOWLIST_ALIAS)
```

Comment:
This could be wrapped into separate function e.g. `get_alias()` and could be used outside as `allowlist.get_alias()` as there are many uses of in tests.